### PR TITLE
Py3k

### PR DESCRIPTION
**Pull request by Azd325**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+skipsdist = True
+
+[testenv]
+deps = pipenv
+commands =
+    python --version
+    pipenv install --dev --skip-lock
+    pytest
+
+
```
Comment (fkoester) [inline tox.ini:13]: The changes regarding tox are not mentioned in the commit message. Are they stongly related to the casting? If not I would put them into a separate commit.

Comment (Azd325) [reply tox.ini:13]: Ich habe es gesplitted.
